src/test/java/org/broadinstitute/hellbender/tools/walkers/annotator/ExcessHetUnitTest.java

```diff
@@ -1,253 +1,292 @@
 package org.broadinstitute.hellbender.tools.walkers.annotator;
 
-import htsjdk.variant.variantcontext.*;
+import htsjdk.variant.variantcontext.Allele;
+import htsjdk.variant.variantcontext.Genotype;
+import htsjdk.variant.variantcontext.GenotypeBuilder;
+import htsjdk.variant.variantcontext.VariantContext;
+import htsjdk.variant.variantcontext.VariantContextBuilder;
 import org.broadinstitute.hellbender.GATKBaseTest;
 import org.broadinstitute.hellbender.utils.variant.GATKVCFConstants;
 import org.broadinstitute.hellbender.utils.variant.GATKVCFHeaderLines;
 import org.testng.Assert;
 import org.testng.annotations.DataProvider;
 import org.testng.annotations.Test;
 
-import java.util.*;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
 
-import static org.mockito.ArgumentMatchers.refEq;
 import static org.mockito.Mockito.mock;
 import static org.mockito.Mockito.when;
 
 public final class ExcessHetUnitTest extends GATKBaseTest {
-    private static double DELTA_PRECISION = .001;
-    private Allele Aref= Allele.create("A", true);
-    private Allele T = Allele.create("T");
-    private Allele C = Allele.create("C");
-    private int[] hetPLs = {240, 0, 240};
-    private int[] homRefPLs= {0, 60, 600};
+    private static final double DELTA_PRECISION = 1E-6;
```

Comment:
Most of the following changes are inconsequential cleanup, but note we do change the precision criteria here.

Reply:
We require more precision now, yes?